packages/studio-ui/src/components/IFramePortal.tsx

```diff
@@ -55,32 +41,3 @@ function useParentDocumentStyles(iframeDocument: Document | undefined) {
     }
   }, [iframeDocument]);
 }
-
-const useCSS = (viewport: Viewport, previewRef: RefObject<HTMLDivElement>) => {
-  const [css, setCss] = useState(
-    (viewport.styles?.width ?? 0) * (previewRef.current?.clientHeight ?? 0) >
-      (viewport.styles?.height ?? 0) * (previewRef.current?.clientWidth ?? 0)
-      ? " w-full "
-      : " h-full "
-  );
-  const handleResize = () => {
-    const tempCss =
-      (viewport.styles?.width ?? 0) * (previewRef.current?.clientHeight ?? 0) >
-      (viewport.styles?.height ?? 0) * (previewRef.current?.clientWidth ?? 0)
-        ? " w-full "
-        : " h-full ";
-    if (tempCss !== css) {
-      setCss(tempCss);
-    }
-  };
-
-  useEffect(() => {
```

Comment:
Just curious, do you know what this was doing before? Since it looks like we were already applying CSS to the iframe.  Other than this question, LGTM!

Reply:
Originally, we were using the aspect ratio instead of the actual pixel sizes to determine the height and width of the viewport. As Alexis [mentioned](https://github.com/yext/studio/pull/336#discussion_r1297305754) in her PR, aspect ratio needs a height or width specified in order to work properly. So, this `useEffect` was in charge of adding either the `w-full` or `h-full` style to the iFrame depending on which viewport dimension we could max out while still fitting within the preview panel size. This hook updated the styling any time the viewport changed or the preview panel's size changed

Reply:
ahh I see.  That makes sense, thanks!